thoth/storages/graph/postgres.py

```diff
@@ -334,43 +341,34 @@ def drop_all(self):
         # Drop alembic version to be able re-run alembic migrations next time.
         self._engine.execute("DROP TABLE alembic_version;")
 
-    def is_schema_up2date(self) -> bool:
-        """Check if the current schema is up2date with the one configured on database side."""
-        import thoth.storages
-        from alembic import config
+    def _get_script_directory_revisions(self):
         from alembic import script
+        alembic_cfg = self._get_alembic_configuration()
+        directory = script.ScriptDirectory.from_config(alembic_cfg)
+        return directory
+
+    def get_script_alembic_version_head(self) -> str:
+        """Get alembic version head from alembic folder scripts."""
+        directory = self._get_script_directory_revisions()
+        head_revision = directory.get_current_head()
+
+        return head_revision
+
+    def get_table_alembic_version_head(self) -> str:
+        """Get alembic version head from database table."""
         from alembic.runtime import migration
 
         if not self.is_connected():
             raise NotConnected("Cannot check schema: the adapter is not connected yet")
 
-        alembic_cfg = config.Config(os.path.join(os.path.dirname(thoth.storages.__file__), "data", "alembic.ini"))
-        alembic_cfg.attributes["configure_logger"] = False
-        alembic_cfg.set_section_option(
-            "alembic", "script_location", os.path.join(os.path.dirname(thoth.storages.__file__), "data", "alembic")
-        )
-        directory = script.ScriptDirectory.from_config(alembic_cfg)
         connection = self._engine.connect()
         context = migration.MigrationContext.configure(connection)
+        current_rev = context.get_current_revision()
+        return current_rev
 
-        database_heads = set(context.get_current_heads())
-        if not database_heads:
-            raise DatabaseNotInitialized("Database is not initialized yet")
```

Comment:
Do we want to keep this exception and the warning below in sources? It might help with tracing issues. Sorry for not being explicit about it above.

Reply:
Yes we can leave it actually. Thanks! re added!